syz-manager/stats.go

```diff
@@ -29,12 +31,40 @@ type Stats struct {
 	corpusCoverFiltered Stat
 	corpusSignal        Stat
 	maxSignal           Stat
+	syzExecTotal        prometheus.GaugeFunc // Prometheus float64.
+	syzCorpusCover      prometheus.GaugeFunc // Prometheus float64.
+	syzCrashTotal       prometheus.GaugeFunc // Prometheus float64.
 
 	mu         sync.Mutex
 	namedStats map[string]uint64
 	haveHub    bool
 }
 
+func (mgr *Manager) initStats() {
+	mgr.stats.syzExecTotal = promauto.NewGaugeFunc(prometheus.GaugeOpts{
+		Name: "syz_exec_total",
+		Help: "Total executions during current execution of syz-manager",
+	},
+		func() float64 { return float64(mgr.stats.execTotal.get()) },
+	)
+	mgr.stats.syzCorpusCover = promauto.NewGaugeFunc(prometheus.GaugeOpts{
+		Name: "syz_corpus_cover",
+		Help: "Corpus coverage during current execution of syz-manager",
+	},
+		func() float64 { return float64(mgr.stats.corpusCover.get()) },
+	)
+	mgr.stats.syzCrashTotal = promauto.NewGaugeFunc(prometheus.GaugeOpts{
+		Name: "syz_crash_total",
+		Help: "Count of crashes during current execution of syz-manager",
+	},
+		func() float64 { return float64(mgr.stats.crashes.get()) },
+	)
+
+	prometheus.Register(mgr.stats.syzExecTotal)
```

Comment:
If we use these variables only here, then do:
```
prometheus.Register(promauto.NewGaugeFunc(prometheus.GaugeOpts{
		Name: "syz_exec_total",
		Help: "Total executions during current execution of syz-manager",
	},
		func() float64 { return float64(mgr.stats.execTotal.get()) },
	))
```

Reply:
Having those variables inside `Stat` struct gives us the ability to replace some of the existing stat counters in the future with these. I am happy to remove those declarations for now and we can add them back in when we start working on that.

Reply:
> I am happy to remove those declarations for now 

Yes, please.
There is nothing more permanent than a temporary solution :)

Reply:
Done :D